stdlib/public/stubs/FoundationHelpers.mm

```diff
@@ -33,26 +33,22 @@
 
 static CFHashCode(*_CFStringHashCString)(const uint8_t *bytes, CFIndex len);
 static CFHashCode(*_CFStringHashNSString)(id str);
-static CFTypeID(*_CFGetTypeID)(CFTypeRef obj);
-static CFTypeID _CFStringTypeID = 0;
+static id(*_CFStringCreateTaggedPointerString)(const uint8_t *bytes, CFIndex numBytes);
+static __swift_uint8_t(*_NSIsNSString)(id arg);
```

Comment:
Using this instead of _CFGetTypeID + _CFStringTypeID lets us keep our dirty data footprint even despite looking up _CFStringCreateTaggedPointerString